src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -201,19 +201,32 @@ public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
                 .collect(Collectors.toList());
         }
 
-        String epimsId = referenceDataServiceHolder
-            .getVenueService()
-            .getEpimsIdForVenue(caseData.getProcessingVenue())
-            .orElse(null);
-
-        Map<String,List<String>> multipleHearingLocations = referenceDataServiceHolder.getMultipleHearingLocations();
-
-        return multipleHearingLocations.values().stream()
-            .filter(listValues ->  listValues.contains(epimsId))
-            .findFirst()
-            .orElseGet(() -> Collections.singletonList(epimsId))
-            .stream().map(epims -> HearingLocation.builder().locationId(epims).locationType(COURT).build())
-            .collect(Collectors.toCollection(ArrayList::new));
+        if (HearingsChannelMapping.isPaperCase(caseData)) {
+            List<String> epimsIds = referenceDataServiceHolder
+                .getVenueService()
+                .getActiveRegionalEpimsIdsForRpc(caseData.getRegionalProcessingCenter().getName());
+
+            return epimsIds.stream()
```

Comment:
(Minor) could perhaps merge the final stream of collect in both paths to remove duplication

Reply:
Done or not, we won't stop deployment on this.